docs-gen/content/rule_set/basics.md

```diff
@@ -84,6 +84,17 @@ SomeBranch.AnotherBranch.MySignalName
 Cabin.Door.Row1.Left.IsLocked
 ```
 
+For string constants used for `default` and `enum`statements for items with string datatype it is recommended to start with `A-Z`
+and then use only `A-Z`, `0-9` and underscore (`_`). Single(`'`) and double (`"`) quotes shall be used only if necessary, and are not needed if the naming recommendation is followed.
+
+
+Examples:
+
+```
+default: UNKNOWN
+enum: [UNKNOWN, ALT_1, ALT_2]
+```
+
```

Comment:
I agree that it would be good if VSS at least states a recommended approach for `allowed`, especially concerning undefined/unknown. I can see two major tracks:

- Recommend that first `allowed` value for strings always shall be `UNKNOWN`. Or e.g. `UNDEFINED` if we consider that a better choice. At least for actuators `UNDEFINED` might be a better option as it could be used to state that you do not care, for instance if someone would like to use something like VSS to specify rental car preferences.
- Recommend that `UNKNOWN` (or similar) generally not shall be used, unless there are good reasons, e.g. compatibility with values from an existing standard/protocol